src/python/doc/rips_complex_user.rst

```diff
@@ -400,12 +400,10 @@ The output is:
 
     [(0, (3.1622776601683795, inf)), (0, (3.1622776601683795, 5.39834563766817)), (0, (3.1622776601683795, 5.39834563766817))]
 
-.. _dtm-rips-complex:
-
 DTM Rips Complex
 ----------------
 
-`DTMRipsComplex <rips_complex_ref.html#dtm-rips-complex-reference-manual>`_ builds a simplicial complex from a point set or a full distance matrix (in the form of ndarray), as described in the above example.
+`DTMRipsComplex (reference manual) <rips_complex_ref.html#dtm-rips-complex-reference-manual>`_ builds a simplicial complex from a point set or a full distance matrix (in the form of ndarray), as described in the above example.
```

Comment:
I would have tried ``:class:`~gudhi.dtm_rips_complex.DTMRipsComplex` ``, but why not.

Reply:
Oh yes I was not thinking of this trick. I used it on 83db295e